micro-rdk-installer/Cargo.toml

```diff
@@ -34,3 +33,4 @@ thiserror.workspace = true
 tokio.workspace = true
 viam-rust-utils.workspace = true
 viam.workspace = true
+esp-idf-part = "0.5.0"
```

Comment:
This should be pulled up to the workspace level `Cargo.toml` and then pulled in here via `esp-idf-part.workspace = true`.